futile.lua

```diff
@@ -0,0 +1,32 @@
+isPhoneOpen = false
+
+CreateThread(function()
+    while true do
+        Wait(5)
+        if isPhoneOpen then
+            DisableControlAction(0, 1, true) -- Look Left/Right
+            DisableControlAction(0, 2, true) -- Look up/Down
+            DisableControlAction(0, 16, true) -- Next Weapon
+            DisableControlAction(0, 17, true) -- Select Previous Weapon
+            DisableControlAction(0, 22, true) -- Jump
+            DisableControlAction(0, 24, true) -- Attack
+            DisableControlAction(0, 25, true) -- Aim
+            DisableControlAction(0, 37, true) -- Weapon Wheel
+        else
+            EnableControlAction(0, 1, true) -- Look Left/Right
```

Comment:
You actually shouldn’t need to enable the control. You’re disabling them in a loop so as long as that loop isn’t running, there is no need to enable them every 1000ms. This current way will likely cause issues with death scripts too.

Reply:
1000ms might be too long if someone closes the phone and tries to reopen it. Would need to test it